test/server/cards/02_SHD/upgrades/PublicEnemy.spec.ts

```diff
@@ -0,0 +1,38 @@
+describe('Public Enemy', function() {
+    integration(function(contextRef) {
+        describe('Public Enemy\'s Bounty ability', function() {
+            it('should give a unit a shield', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['battlefield-marine', 'atst']
+                    },
+                    player2: {
+                        groundArena: [
+                            { card: 'wampa', upgrades: ['public-enemy'] },
+                            { card: 'escort-skiff' }
+                        ]
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.atst);
+
+                // Destroys Wampa
+                context.player1.clickCard(context.wampa);
+
+                // Damange from Wampa to be assigned
+                expect(context.atst.damage).toBe(4);
```

Comment:
Can skip this check since it's not really related to the card ability being tested